scripts/rtl872x.tcl

```diff
@@ -107,7 +108,8 @@ proc openocd_print_memory {address length} {
     init
     reset halt
     set value ""
-    mem2array value 8 $address $length
+    set value [read_memory $address 8 $length]
```

Comment:
I'm pretty sure this one is going to fail. Because the way we're handling the value object in the follow lines.
Now the returned object is pretty different from the deprecated function